ingeniamotion/configuration.py

```diff
@@ -175,6 +177,9 @@ def set_max_acceleration(self, acceleration, servo=DEFAULT_SERVO,
             servo (str): servo alias to reference it. ``default`` by default.
             axis (int): servo axis. ``1`` by default.
         """
+        self.logger.warning('This method is deprecated. '
```

Comment:
Probably the user won't know what function emit this log, add the function name in it.